code/modules/uplink/uplink_items.dm

```diff
@@ -990,39 +1007,112 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 	cost = 5
 	restricted = TRUE
 
+/datum/uplink_item/explosives/doorcharge
+	name = "Airlock Charge"
+	desc = "A small explosive charged that can be placed into the wiring of any airlock. \
+			The moment someone attempts to open the airlock, the charge will detonate, destroying the airlock and injuring anyone nearby."
+	item = /obj/item/doorCharge
```

Comment:
Please nerf this a bit to not use dev as its a sleep/auto kill for even armord targets, place 2 on sec and bam you killed off all of sec or a head